packages/fast-usdc/src/fast-usdc.contract.js

```diff
@@ -167,7 +166,17 @@ export const contract = async (zcf, privateArgs, zone, tools) => {
     async makeWithdrawFeesInvitation() {
       return poolKit.feeRecipient.makeWithdrawFeesInvitation();
     },
-    async connectToNoble() {
+    /**
+     * @param {string} agoricChainId
+     * @param {string} nobleChainId
+     * @param {IBCConnectionInfo} agoricToNoble
+     */
+    async connectToNoble(agoricChainId, nobleChainId, agoricToNoble) {
+      trace('connectToNoble', agoricChainId, nobleChainId, agoricToNoble);
+      chainHub.updateConnection(agoricChainId, nobleChainId, agoricToNoble);
+      // v1 has `NobleAccount` which we don't expect to ever settle.
+      const nobleAccountV = zone.makeOnce('NobleICA', () => makeNobleAccount());
```

Comment:
Nice. I believe the definition is fine here - outside of the first crank - since we've already defined the kind (a `Vow`)

Question - we _sort of_ throw away `nobleAccount` in the proceeding line. Knowing we likely need this handy to support reaching non-cosmos destinations via CCTP, i wonder if we should put it somewhere more reachable. Maybe baggage?

Reply:
This does put it in baggage.

`zone` comes from `makeDurableZone(baggage) (and then a subzone) in `provideOrchestration`.

Reply:
Ok - for a gap in my understanding - what does retrieving look like? Is it simply `baggage.get('NobleICA')`?

Reply:
close... the `get` would have to be on the collection that backs the `contract` subzone.

`zone.makeOnce('NobleICA', () => assert.fail('known to be present'))` would certainly do it.

I'd have to pore over the implementation of `subZone` and `makeOnce` to see if there's something more straightforward... maybe `baggage.get('contract').get('NobleICA')`.